test/distributed/_shard/checkpoint/test_file_system_checkpoint_cpu.py

```diff
@@ -0,0 +1,542 @@
+# Owner(s): ["oncall: distributed"]
+
+import sys
+import os
+import shutil
+import tempfile
+from typing import Dict, cast
+
+import torch
+import torch.distributed as dist
+from torch import Tensor
+from torch.distributed._shard import sharded_tensor
+from torch.distributed._shard.sharded_tensor import ShardedTensor, state_dict_hook
+from torch.distributed._shard.sharding_spec import (
+    ChunkShardingSpec,
+    EnumerableShardingSpec,
+    ShardingSpec,
+    ShardMetadata,
+)
+from torch.testing._internal.common_utils import TestCase
+from torch.testing._internal.distributed._shard.sharded_tensor import (
+    ShardedTensorTestBase,
+    with_comms,
+)
+from torch.testing._internal.distributed._shard.sharded_tensor._test_st_common import (
+    MyShardedModel1
+)
+
+
+from torch.testing._internal.common_utils import (
+    TEST_WITH_DEV_DBG_ASAN,
+    run_tests,
+)
+
+from torch.distributed._shard.checkpoint import (
+    FileSystemReader,
+    FileSystemWriter,
+    load_state_dict,
+    save_state_dict,
+)
+
+
+if TEST_WITH_DEV_DBG_ASAN:
+    print(
+        "Skip dev-asan as torch + multiprocessing spawn have known issues",
+        file=sys.stderr,
+    )
+    sys.exit(0)
+
+def _sharded_tensor_gather(
```

Comment:
https://github.com/pytorch/pytorch/blob/394aca9a26cd760d1f761821452b0d308ac35b03/torch/distributed/_shard/sharded_tensor/api.py#L269

This method already been merged by @rodrigobdz so I guess we can safely remove this.